server/project.go

```diff
@@ -263,7 +263,7 @@ func (s *Server) registerProjectRoutes(g *echo.Group) {
 				URL:                    fmt.Sprintf("%s/%s/%s", s.profile.ExternalURL, gitlabWebhookPath, repositoryCreate.WebhookEndpointID),
 				SecretToken:            repositoryCreate.WebhookSecretToken,
 				PushEvents:             true,
-				PushEventsBranchFilter: repositoryCreate.BranchFilter,
+				PushEventsBranchFilter: "",    // For consistency with GitHub, we leave blank for all.
```

Comment:
Should we also change the frontend request body? @LiuJi-Jim

Reply:
No, only backend should be changed.